packages/pacstall-gui-git/pacstall-gui-git.pacscript

```diff
@@ -12,17 +12,18 @@ pkgver() {
 }
 
 prepare() {
-  sudo mkdir -p "${pkgdir}/usr/bin" "${pkgdir}/usr/lib/pacstall-gui" "${pkgdir}/usr/share/applications"
+  sudo mkdir -p "${pkgdir}/usr/bin" "$homedir/.config/pacstall-gui" "${pkgdir}/usr/share/applications"
```

Comment:
Do not put `$homedir` anything in any functions except post ones.

Reply:
Is the `settings.sh` file still available in post install if it is not put anywhere else though?